containers/cpp/.devcontainer/base.Dockerfile

```diff
@@ -8,6 +8,20 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install build-essential cmake cppcheck valgrind clang lldb llvm gdb \
     && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
 
+# Install additional packages needed by vcpkg: https://github.com/microsoft/vcpkg/blob/master/README.md#installing-linux-developer-tools
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends tar curl zip unzip pkg-config bash-completion ninja-build
```

Comment:
Are all these tools needed for vcpkg? Should be part of the install script instead? I'm just trying to scope the tooling for a specific need.

Reply:
yes, thx, done